packages/ubuntu_desktop_installer/test/product_info_extractor_test.dart

```diff
@@ -0,0 +1,105 @@
+import 'package:file/file.dart';
+import 'package:file/memory.dart';
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:ubuntu_desktop_installer/utils/product_info_extractor.dart';
+
+void main() {
+  group('ProductInfoExtractor', () {
```

Comment:
All the tests are in the same group, so this is an unnecessary level.

Reply:
`group` is necessary for `setUp()` method. it's also good convention

Reply:
It's not necessary. See https://pub.dev/packages/test :

> You can use the setUp() and tearDown() functions to share code between tests. The setUp() callback will run before every test _**in a group or test suite**_, and tearDown() will run after.

It doesn't hurt though, feel free to keep it.